### ISSUE TITLE
Sprint 4 (10.22.2024- 11.4.2024)

### ISSUE THREAD

**Issue by meganhicks** [department-of-veterans-affairs/abd-vro#3511]
**Sprint 4**

# Capacity

The percentages below reflect the amount of capacity the team members have to work towards sprint goals and priorities. Therefore, the sprint plan commitments should be reduced using these percentages. Note: includes time off, non-sprint commitments, and new team member ramp-up.

All Eng: 69.5% (14 pts)
All UX: 91.8% (3.7 pts)
All Team: 73.6% (17.6 pts)

Capacity notes from Berni: Julius and Geoffrey are starting on VRO on 10/28, but won't be taking on sprint work until the next sprint. I've added some capacity for them to pair with others on tickets. 
Ponnia out 10/22; Derek 70% on VRO; Katelyn 33% on VRO.

## Carry-Over Work

- [ ] Understand CC query use cases #3553
- [ ] Alerting rabbitmq metrics in datadog #3482
- [x] Structure Single Page App for Streamlit #3554
- [x] Implement Shared ArgoCD Secrets for our applications #3614
- [x] Update Datadog documentation on the wiki #3563
- [x] Dependency Management Page on wiki #3036

# Sprint Plan 

### Silent Failures (Epic #2918)

This should be finished!

### Data Visibility (Epic #2918)

- [ ] Deploy Streamlit to prod #3613 - blocked by LHDI
- [ ] Connect Streamlit to RDS #3557
- [x] Create a Generic user for Streamlit #3555
- [ ] Rollover: Understand CC query use cases #3553

### Improve the Partner Team Experience (Epic #2961)

Nothing this Sprint should be focused on Streamlit 

### Keep the platform running (Epic #2806)

- [x] Streamline the VRO Engineer Onboarding Doc #3592
- [ ] Consolidate all on-call data and instructions in one location #3519
- [ ] LHDI SDE countermeasure: WASA scan #3568

### Metrics (Epic #3190)

- [ ] Define how we will measure Impact to Veterans/VSRs/RVSRs based on tools built on VRO #3192

### Improve Deployments (Epic #2615)

- [x]  Implement Shared ArgoCD Secrets for our applications #3614
- [x]  Secrets Automation Cleanup #3540

Stretch Goal: 
- [ ] Determine where certification management is manual #3324

### Who is On Call?
- [x] #3676
- Primary: Kyle
- Secondary: Gabriel 

### Demo Plan

- What could we demo?

# Unplanned Work (these tickets were added post-Sprint Planning):
- [x] create RBAC roles for service account #3657
- [x] Partner Team Production Deployment workflow - Leftover #3626

**Comment by BerniXiongA6** [comment]
Hey VRO Team (@dfitchett @katelyncai @brostk @bianca-rivera @Ponnia-M @PaulKBaumann @gabezurita )! Despite the unexpected news of the decision to decommission VRO platform and halt all current initiatives, you all finished Sprint 4 very strong. The sprint closed at 122% of team capacity (21.5 pts / 17.6 pts) and 100% of sprint plan (21.5 pts / 21.5 pts). You should all be very proud of yourselves. cc: @meganhicks @amylai-va
